Probesys/bileto
# new: Allow to renew the contracts

## Related issue(s)



https://github.com/Probesys/bileto/issues/25

## How to test manually



- create a contract, name it something like "Contract 2023"
- go to the contract and click on "renew"
- check the name is "Contract 2024" and that it starts the day after the last day of the initial contract
- check also that the other information are identical

## Checklist
